package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "powerbi-visuals-asterplot",
   "description": "An Aster plot is a twist on a standard donut chart, using a second value to drive sweep angle.",
-  "version": "0.5.7",
+  "version": "1.0.1",
   "author": {
     "name": "Microsoft",
     "email": "pbicvsupport@microsoft.com"
   },
   "scripts": {
-    "postinstall": "typings install && pbiviz update 1.3.0",
-    "typings": "typings",
+    "postinstall": "pbiviz update 1.7.0",
     "pbiviz": "pbiviz",
```

Comment:
Please use `"lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src|test)/**/*.ts\"",` instead of existing one